express/app/models/user.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      lastName: DataTypes.STRING,
+      email: DataTypes.STRING,
+      password: DataTypes.STRING
+    },
+    {}
+  );
+  User.associate = function(models) {
+    // associations can be defined here
+  };
+  return User;
```

Comment:
We define models in lower case and plural. `sequelize.define('users' ...` instead of `sequelize.define('User' ...`.